Generate keys (#73)
ggxchain/documentation: src/README.md

### What is GGX chain?

> GGX chain is the Interchain Infrastructure Protocol that utilizes Multi-Party Computation for interoperable communication and advanced liquidity routing over a network of decentralized independent sub-chains.

Reviewer: ```suggestion
GGX chain is the interchain infrastructure protocol that utilizes Multi-Party Computation (MPC) for interoperable communication and advanced liquidity routing over a network of decentralized independent sub-chains.
```

GGX chain is built to connect arbitrary chains with each other turning sharded environment into the internet of blockchains.

### Why GGX chain?

> GGX chain is a single platform where you can access most of available blockchain (aiming to provide availability for all valueable chains). GGX chain utilizes hybrid virtual machine, so you can program in Solidity as well as any general purpose programming language that can be compiled into the WASM (e.g. Rust, Javascript).

Reviewer: ```suggestion
GGX chain is an interchain communication platform, allowing you to send messages and liquidity from one blockchain to another. GGX chain is also a smart contract platform, utilizing a hybrid virtual machine. GGX chain's hybrid virtual machine allows users to program in Solidity as well as any general purpose programming language that can be compiled into the WASM (e.g. Rust, Javascript).
```

<div align="center">
  <img src="assets/logo.png" class="logo-bright" alt="GGX Chain logo" />
  <img src="assets/logo-dark.png" class="logo-dark" alt="GGX Chain logo" />
</div>

> ### What is GGX chain?

Reviewer: ```suggestion
### What is GGX Chain?
```

GGX chain is the interchain infrastructure protocol that utilizes Multi-Party Computation (MPC) for interoperable communication and advanced liquidity routing over a network of decentralized independent sub-chains.

GGX chain is built to connect arbitrary chains with each other turning sharded environment into the internet of blockchains.

> ### Why GGX chain?

Reviewer: ```suggestion
### Why GGX Chain?
```
